modules/ml/include/opencv2/ml.hpp

```diff
@@ -1164,6 +1164,14 @@ class CV_EXPORTS_W RTrees : public DTrees
      */
     CV_WRAP virtual Mat getVarImportance() const = 0;
 
+    /** Returns the result of each individual tree in the forest.
+    In case the model is a regression problem, the method will return each of the tree's
+    result for each of the sample cases. If the model is a classifier, it will return 
+    a Mat with samples + 1 rows, where the first row gives the class number and the 
+    following rows return the votes each class had for each sample.
+    */
+    CV_WRAP virtual void getVotes(InputArray, OutputArray, int) {};
```

Comment:
ABI compatibility check still fails here, so I think It's better to create an implicit implementation. Here is an example how to call method of child `Impl` class:
```
Mat SVM::getUncompressedSupportVectors() const
{
    const SVMImpl* this_ = dynamic_cast<const SVMImpl*>(this);
    if(!this_)
        CV_Error(Error::StsNotImplemented, "the class is not SVMImpl");
    return this_->getUncompressedSupportVectors_();
}
```